doc/mpl_toolkits/index.rst

```diff
@@ -172,6 +172,26 @@ level interface for drawing statistical graphics with matplotlib. It
 aims to make visualization a central part of exploring and
 understanding complex datasets.
 
+.. image:: /_static/seaborn.png
+    :height: 157px
+
+.. _toolkit_holoviews:
+
+holoviews
+=========
+(*Not distributed with matplotlib*)
+
+`holoviews <http://holoviews.org>`_ makes it easier to visualize data
```

Comment:
I seem to recall @tacaswell saying something about these links requiring two `_` at the end.